.github/workflows/docs.yml

```diff
@@ -19,19 +22,17 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: Setup .NET
-      uses: actions/setup-dotnet@v3
+      uses: actions/setup-dotnet@v3.0.3
```

Comment:
Why are we specifying a specific version?

Reply:
i-
i have no idea why i did that